exercises/9_array_hash.rb

```diff
@@ -0,0 +1,19 @@
+def arr_hash(arr)
+  hash = Hash.new()
+
+  arr.sort_by! {|el| el.length}
+
+  arr.each do |el|
+    key = el.length
+    if(hash[key])
```

Comment:
You can do something like this also:
hash[key] ||= []
hash[key] << element
It does the same thing, saves some line of code.